ddtrace/contrib/pymongo/patch.py

```diff
@@ -18,7 +17,7 @@
 config._add(
     "pymongo",
     dict(
-        _default_service=mongox.SERVICE,
+        _default_service="pymongo",
```

Comment:
Do we also need to update https://github.com/DataDog/dd-trace-py/blob/1.x/ddtrace/contrib/pymongo/client.py#L66? 

Also changing the default service name from `mongodb` to `pymongo` is a breaking change in our library. Do we need to this change in this PR?

Reply:
👍 The tests will always fail given that the service name will then always default to ```monogdb``` instead of ```pymongo```. We could maybe have it so that mongodb would accept custom values since ```pymongo``` seems to be taking the service name from mongo? Similar to what we did for rediscluster here: https://github.com/DataDog/dd-trace-py/pull/4059/commits/1c12dec21194de84d452fe750ef331a24efacb78

Reply:
Should we remove `ddtrace.ext.mongo.SERVICE`?

Reply:
you mean deprecate and not remove correct?

Reply:
It seems like ```mongoengine``` is the only other integration that uses ```ddtrace.ext.mongo.SERVICE```, we could  have it manually use ```mongo`` for the service name or provide it with it's own dedicated service name i.e. ```mongoengine``` if we are going to deprecate it.

Reply:
I was just curious. We can avoid deprecating it for now